source/migrate.txt

```diff
@@ -1,60 +0,0 @@
-=====================================
-Migrate Your App from Stitch to Realm
```

Comment:
I wonder if this page and the one below it - stitch.txt - should continue to exist? I don't know how long we plan/need to support migrating from Stitch to Realm.

Reply:
This one exists on the Realm side so I figured we could redirect.